circle.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
 
 test:
   override:
-    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint
+    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint bin/cloud-login
```

Comment:
https://github.com/koalaman/shellcheck/wiki/Directive can be added to ignore specific errors for specific lines.